pkg/rpc/wrappers/contract_state.go

```diff
@@ -0,0 +1,58 @@
+package wrappers
+
+import (
+	"github.com/CityOfZion/neo-go/pkg/core/state"
+	"github.com/CityOfZion/neo-go/pkg/smartcontract"
+	"github.com/CityOfZion/neo-go/pkg/util"
+)
+
+// ContractState wrapper used for the representation of
+// state.Contract on the RPC Server.
+type ContractState struct {
+	Version     byte                      `json:"version"`
+	ScriptHash  util.Uint160              `json:"hash"`
+	Script      []byte                    `json:"script"`
+	ParamList   []smartcontract.ParamType `json:"parameters"`
+	ReturnType  smartcontract.ParamType   `json:"returntype"`
+	Name        string                    `json:"name"`
+	CodeVersion string                    `json:"code_version"`
+	Author      string                    `json:"author"`
+	Email       string                    `json:"email"`
+	Description string                    `json:"description"`
+	Properties  Properties                `json:"properties"`
+}
+
+// Properties response wrapper.
+type Properties struct {
+	HasStorage       bool `json:"storage"`
+	HasDynamicInvoke bool `json:"dynamic_invoke"`
+	IsPayable        bool `json:"is_payable"`
+}
+
+// NewContractState creates a new Contract wrapper.
+func NewContractState(c *state.Contract) ContractState {
+	scriptHash, err := util.Uint160DecodeBytesBE(c.ScriptHash().BytesLE())
```

Comment:
I'm not sure what are you doing here, `c.ScriptHash()` is already an `Uint160`, if you need to reverse the byte order it has a `Reverse` method.

Reply:
Done: used `c.ScriptHash().Reverse()`